internal/kafka/command_topic_describe_onprem.go

```diff
@@ -69,94 +50,17 @@ func (c *command) describeOnPrem(cmd *cobra.Command, args []string) error {
 }
 
 func DescribeTopic(cmd *cobra.Command, restClient *kafkarestv3.APIClient, restContext context.Context, topicName, clusterId string) error {
-	// Get partitions
-	partitions, resp, err := restClient.PartitionV3Api.ListKafkaPartitions(restContext, clusterId, topicName)
-	if err != nil {
-		return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
-	} else if partitions.Data == nil {
-		return errors.NewErrorWithSuggestions(errors.InternalServerErrorMsg, errors.InternalServerErrorSuggestions)
-	}
-
-	topic := &TopicData{
-		TopicName:      topicName,
-		PartitionCount: len(partitions.Data),
-		Partitions:     make([]*PartitionData, len(partitions.Data)),
-	}
-
-	for i, partition := range partitions.Data {
-		// For each partition, get replicas
-		replicas, resp, err := restClient.ReplicaApi.ClustersClusterIdTopicsTopicNamePartitionsPartitionIdReplicasGet(restContext, clusterId, topicName, partition.PartitionId)
-		if err != nil {
-			return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
-		} else if replicas.Data == nil {
-			return errors.NewErrorWithSuggestions(errors.InternalServerErrorMsg, errors.InternalServerErrorSuggestions)
-		}
-		topic.Partitions[i] = &PartitionData{
-			TopicName:              topicName,
-			PartitionId:            partition.PartitionId,
-			ReplicaBrokerIds:       make([]int32, len(replicas.Data)),
-			InSyncReplicaBrokerIds: make([]int32, 0, len(replicas.Data)),
-		}
-		for j, replica := range replicas.Data {
-			if replica.IsLeader {
-				topic.Partitions[i].LeaderBrokerId = replica.BrokerId
-			}
-			topic.Partitions[i].ReplicaBrokerIds[j] = replica.BrokerId
-			if replica.IsInSync {
-				topic.Partitions[i].InSyncReplicaBrokerIds = append(topic.Partitions[i].InSyncReplicaBrokerIds, replica.BrokerId)
-			}
```

Comment:
I thought about this a bit more and ultimately, I think we don't need to add `Leader`, `Replicas` or `ISR` to `confluent kafka replica list` because `confluent kafka replica list --topic <id> --partition <id>` makes it redundant.

Using these flags:
- `Leader`: In `replica list`, this is the broker ID where `Leader` (a boolean in `replica list`) is `true`.
- `Replicas`: In `replica list`, this is all the values in the `Broker ID` column.
- `ISR`: In `replica list`, it's all broker IDs where `In ISR` is `true`.